nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/resources/docs/org.apache.nifi.processors.azure.storage.PutAzureDataLakeStorage/additionalDetails.html

```diff
@@ -28,27 +28,29 @@
 
 <h3>File uploading and cleanup process</h3>
 
-<h4>New file</h4>
+<h4>New file upload</h4>
 
 <ol>
-    <li>An empty file is created.</li>
-    <li>Content is appended to file.</li>
-    <li>In case append failure the file is deleted.</li>
-    <li>In case file deletion failure the empty file remains on the server.</li>
+    <li>A temporary file is created with random prefix under the given path in '_$azuretempdirectory$'.</li>
```

Comment:
I think this changed to `_nifitempdirectory` in the final commits.
```suggestion
    <li>A temporary file is created with random prefix under the given path in '_$nifitempdirectory$'.</li>
```